R/orderly.R

```diff
@@ -1,5 +1,5 @@
 ##' List source reports - that is, directories within `src/` that
-##' contain a file `orderly.R`
+##' contain a file `<reportname>.R`
```

Comment:
I think this could be reworded or explained better

```suggestion
##' look suitable for running with orderly; these will be directories
##' that contain a `.R` file with the same name as the directory
##' (e.g., `src/data/data.R` corresponds to `data`).
```

More of a mouthful, but perhaps clearer?

Reply:
yh definitely clearer, ive altered it slightly, have said it will list dirs that contain an entrypoint file - a `.R` file with the same name...